win32/src/win32evtlog.i

```diff
@@ -577,6 +577,8 @@ static LPWSTR FormatMessageInternal(EVT_HANDLE metadata, EVT_HANDLE event, DWORD
 		char buf[2048];
 		sprintf(buf, "EvtFormatMessage: allocated %d, need buffer of size %d", allocated_size, returned_size);
 		PyWin_SetAPIError(buf, err);
+		free(buf);
```

Comment:
This isn't freeing the allocated buffer, because `buf` is redefined as an array on the stack within this scope, so you'll be freeing the stack variable!

Maybe rename `char buf[2048]` to to `char errorMessage[2048]` and then have `sprintf` and `PyWin_SetAPIError` use `errorMessage` rather than `buf`.

This way `buf` will still refer to the `buf` defined in the outer scope.

Reply:
Yikes sorry about that. Fixed, thank you!